test/file.js

```diff
@@ -96,10 +96,10 @@ describe('File', function() {
       done();
     });
 
-    it('defaults path to null', function(done) {
+    it('defaults path to undefined', function(done) {
       var file = new File();
-      expect(file.path).toNotExist();
-      expect(file.path).toEqual(null);
+      expect(file.path).toBeFalsy();
+      expect(file.path).toEqual(undefined);
```

Comment:
Not sure if we should be changing this test or changing the implementation 🤔

Reply:
Because this assertion was failed when expected value is `null`.
The `file.path` was `undefined` before this PR, but `expect`'s `toEqual` was not strict.

Reply:
I've updated the code.